### DISCUSSION TITLE
Fill the empty info SrcGeoCity & SrcGeoState ?

### DISCUSSION THREAD

**Discussion by kronicklez** [akvorado/akvorado#1491] [Q&A]
Hi @vincentbernat / expert,

May i know what parameter need to put to fill empty info SrcGeoCity & SrcGeoState in the akvorado .yml. Is ir just "city" and "state" only?

name: Test
role: customers
site: Test
region: Test
tenant: Test

Thanks and appreciate your feedback

**Comment by vincentbernat** [comment]
Yes.
